pkg/kubectl/resource/builder.go

```diff
@@ -162,6 +162,21 @@ func NewBuilder(clientConfigFn ClientConfigFunc, restMapper meta.RESTMapper, cat
 	}
 }
 
+func NewBuilder(restClientGetter RESTClientGetter) *Builder {
+	restMapper, err := restClientGetter.ToRESTMapper()
```

Comment:
One small nit is to use the error names like in the previous code, iow. `mapperError` and `clientError`, only for readability reasons :wink: